helper_lib.sh

```diff
@@ -88,5 +88,11 @@ get_docker_effective_command_line_args() {
 
 get_systemd_service_file(){
     SERVICE="$1"
-    systemctl show -p FragmentPath "$SERVICE" | sed 's/.*=//'
+
+    systemctl show -p FragmentPath "$SERVICE" 2> /dev/null 1>&2
+    if [ $? -eq 0 ]; then
+        systemctl show -p FragmentPath "$SERVICE" | sed 's/.*=//'
+    else
+        echo "/usr/lib/systemd/system/$SERVICE"
+    fi
```

Comment:
I think this missing the case when the following are true
1) docker-bench running inside a Centos7 container on a CentOS7 Host
2) The host does have a SystemD override file present at /etc/systemd/system/${SERVICE}

I'm not so familiar with systemd and where overrides can live. Would there be a drawback to avoiding use of systemctl and doing "```if [ -e  /etc/systemd/system/${SERVICE} ]  then echo ... else ...```fallback to /usr/lib/system/system/${SERVICE} ?